src/components/Book.js

```diff
@@ -1,11 +1,17 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
+const Book = ({ book, delete: handleRemoveBook }) => (
   <table className="table">
     <tr>
-      <td>{book.id}</td>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
+      <td className="col-3">{book.id}</td>
+      <td className="col-3">{book.title}</td>
+      <td className="col-3">{book.category}</td>
+      <td className="col-3">
+        <button type="button" onClick={() => handleRemoveBook(book)}>
```

Comment:
Please implement the handRemoveBook function in the BookList component. This method should only be passed as a prop in this component.